openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -775,10 +775,11 @@ Content-Length: 0
 The Credential Issuer provides a nonce value in the HTTP response with a 2xx status code and the following parameters included as top-level members in the message body of the HTTP response using the application/json media type:
 
 * `c_nonce`: REQUIRED. String containing a nonce to be used when creating a proof of possession of the key proof (see (#credential-request)).
-* `c_nonce_expires_in`: OPTIONAL. Number denoting the lifetime in seconds of the `c_nonce`. This value serves only as a hint to the Client, indicating how long the Credential Issuer is likely to accept the `c_nonce` as valid.
 
 Due to the temporal and contextually sensitive nature of the `c_nonce` value, the Credential Issuer MUST make the response uncacheable by adding a `Cache-Control` header field including the value `no-store`.
 
+A wallet SHOULD assume that the returned `c_nonce` value remains valid and continue using it in credential requests (see [#credential-request]) until the credential endpoint returns an `invalid_nonce` error response.
```

Comment:
```suggestion
A Wallet SHOULD continue using the returned `c_nonce` value in credential requests (see [#credential-request]) until the Credential endpoint returns an `invalid_nonce` error response.
```

wallet -> Wallet

I have suggested a reformulation of the sentence, as I am not a big fan of assumptions, especially when they are not intended for sentient beings but for devices such as wallets.